docs/limitations.md

```diff
@@ -242,12 +242,12 @@ See more documentation at
 
 #### `docker --device`
 
-host device support (`docker run --device`) is not supported. This could
-be mapped to the quite similar QEMU `-device` option to enact device
-pass-through if that particular device is supported by VFIO, but the QEMU
-option requires precise knowledge of which sort of device is being
-mapped, and that information is not necessarily easily available from
-the docker `--device` command.
+Support has been added to pass devices using VFIO passthrough.
+Devices that support IOMMU can be assigned to vfio-pci driver and passed
+on the docker command line with --device=/dev/vfio/$(iommu_group_number) 
+where iommu_group_number is the IOMMU group that the device belongs to.
+Support for passing other devices including block devices with --device
+has not been added yet.
```

Comment:
Can you mention that if there are multiple devices in the same group in the current implementation all of them will get assigned to CC.

Reply:
Done